src/bartiq/_routine.py

```diff
@@ -412,6 +412,9 @@ class Port(BaseModel):
     size: Optional[AnnotatedValue]
     meta: Optional[dict[str, Any]] = Field(default_factory=dict)
 
+    def __repr__(self):
+        return f'{self.__class__.__name__}({self.parent.name}.#{self.name}, ({self.direction}) size="{self.size}")'
+
```

Comment:
That's a great point! If `self.parent` is `None`, I feel like something needs to indicate that the port is orphaned. Otherwise, I think it is not obvious that it is orphaned as it would look like:
```
Port(.#In_0, (input) size="R")
```
WDYT of the following options?
```
1.) Port({}.#In_0, (input) size="R")
2.) Port(_orphan_.#In_0, (input) size="R")
3.) Port(_.#In_0, (input) size="R")
4.) Port(none.#In_0, (input) size="R")
```

Reply:
I think that 4. one is the best. @mstechly any thoughts?

Reply:
Yeah, 4 looks good.
But actually, I think I'd like `Port(none.#In_0, size="R", input)` more than `Port(none.#In_0, (input) size="R")`, WDYT guys?

Reply:
I pushed the update, but noticed one more thing. If `self.size` is `None`, should the output be `Port(none.#In_0, size="None", input)` or `Port(none.#In_0, size=None, input)`? I realize this is getting very specific and detailed, but I figure might as well do this once and not have to worry about it again.

Reply:
Without quotes looks better for me, as quotes would suggest it's a string that happens to say "None".